code/mission/missionmessage.cpp

```diff
@@ -316,7 +316,8 @@ int add_avi( char *avi_name )
 	strcpy_s( extra.name, avi_name );
 	strcpy_s( extra.anim_data.filename, avi_name);
 	extra.num = -1;
-	generic_anim_load(&extra.anim_data);
+	generic_anim_load(&extra.anim_data);   // load only to validate the anim
+	generic_anim_unload(&extra.anim_data); // unload to not waste bmpman slots
```

Comment:
Does the current code display a warning if an EFF head ani doesn't exist? As far as I can tell it doesn't but that's a pretty important part of validating the table entry.